kong/core/balancer.lua

```diff
@@ -173,7 +173,7 @@ local get_balancer = function(target)
   if not balancer then
     -- no balancer yet (or invalidated) so create a new one
     balancer, err = ring_balancer.new({
-        wheelsize = upstream.slots,
+        wheelSize = upstream.slots,
```

Comment:
Yes, you are right, just double checked it. I ran into it when ripping out the `orderlist` property. But it seems that that change actually made this bug surface, as it was hidden by the fact that the orderlist was always given.